cmd/argo/commands/clustertemplate/create.go

```diff
@@ -25,6 +25,19 @@ func NewCreateCommand() *cobra.Command {
 	command := &cobra.Command{
 		Use:   "create FILE1 FILE2...",
 		Short: "create a cluster workflow template",
+		Example: `# Create a Cluster Workflow Template with Default Options:
+		argo cluster-template create FILE1.yaml
+
+		# Create a Cluster Workflow Template in JSON Format:
+		argo cluster-template create FILE1.yaml --output json 
+
+		# Create a Cluster Workflow Template with Relaxed Validation (default true):
```

Comment:
```suggestion
		# Create a Cluster Workflow Template with relaxed validation:
```

- The options already list the defaults, so we don't need to repeat them
- "Relaxed Validation" should not be capitalized as it is not a name / proper noun